ovm20-temp/docker-compose-ovm20.yml

```diff
@@ -0,0 +1,211 @@
+# TODO: Prefix all env vars with service name
```

Comment:
Why does this live here? I'd rather do this the right way and not need to include a docker compose file in our simple getting started tutorial